### DISCUSSION TITLE
How can I use the Command key in a keybinding?

### DISCUSSION THREAD

**Discussion by jsjoeio** [LunarVim/LunarVim#3547] [Q&A] [answered]
I've tried a couple different things but haven't had much luck. I searched other discussions and issues but didn't see much either. 

I thought this would work: 

```lua
lvim.keys.normal_mode["<D-w>"] = ":BufferKill<CR>"
```

What am I doing wrong?

**Comment by danielo515** [comment]
I have very simillar ones. The syntax of the right hand side is correct. You can either use `:` or `cmd`:

```lua
-- This all work
lvim.keys.normal_mode["<S-h>"] = ":BufferLineCyclePrev<CR>"
lvim.keys.normal_mode["<C-F>"] = "<cmd>lua require('user.telescope').grep_files()<cr>"
lvim.keys.normal_mode["<S-X>"] = "<cmd>BufferKill<cr>"
```

However, I don't know what that `D-W` is. What keys do you want to bind?

**Reply by jsjoeio** [reply]
Oh nice! Well glad I was 50% correct. I was trying to map it to Command (macOS super key) + W. Do you know how to do that?

**Comment by danielo515** [comment, marked as answer]
Command is not availabe inside terminal. You need to configure you terminal
emulator to send a specific key combination and map to that. For example,
in kitty you can map it to alt and then use M- maps

**Reply by jsjoeio** [reply]
OHH! Okay, i'll try that instead. Thank you!

**Reply by danielo515** [reply]
Let me know if you need some examples. I have kitty bindings in my public dotfiles
